src/main/java/com/google/sps/data/WaterContaminant.java

```diff
@@ -0,0 +1,44 @@
+package com.google.sps.data;
```

Comment:
I dont know you intended to have all the lines commented out. But if that was your intention, i think it would be best to leave it for another PR.